src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,59 @@
 import React from 'react';
-
+import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
+import { Url } from 'url';
+
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({
+  movie: {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+    imdbId,
+  },
+}) => (
```

Comment:
Remove the destructuring here and do it inside the component
```suggestion
export const MovieCard: React.FC<Props> = ({ movie }) => (
```